host_vars/OCT-CORE-2/interfaces.yaml

```diff
@@ -49,6 +59,21 @@ interfaces:
   1/3:
     state: "down"
   1/4:
+    fanout: "quad"
+    fanout_speed: "25G"
+  1/4/1:
+    description: "ESI-CTRL-U33-R"
+    state: "up"
+    mtu: 9216
+    untagged: 214
+    tagged:
+      - 911
+      - 207
+  1/4/2:
+    state: "down"
+  1/4/3:
+    state: "down"
+  1/4/4:
```

Comment:
Are we using 1/4 already or will this be a new connection with a new cable? In the old manifest it was down.

Reply:
I made a mistake, it should be 1/1/4. I'll update it.